docs/source/configuration.rst

```diff
@@ -1348,6 +1348,13 @@ notification_icon
 This option specifies which icon is shown in HTML5 notifications, as provided
 by the ``src/converse-notification.js`` plugin.
 
+notify_nicknames_without_references
+-----------------------------------
+
+* Default: ``false``
+
+Enables desktop notification when user nickname is mentioned without the ``@`` sign.
```

Comment:
This text isn't precise enough.

 It's not really about the `@` sign. It's about whether an incoming stanza includes [XEP-372](https://xmpp.org/extensions/xep-0372.html) references that maps JIDs to nicknames.

Converse allows the user to use the `@` sign add references, but other XMPP clients don't, and the `@` sign isn't event sent over the wire.

Alternative suggestion:

```
Enables notifications for nicknames in messages that don't have associated XEP-0372 references linking them to the JID of the person being mentioned. In Converse, these would be nicknames that weren't mentioned via the ``@`` sign.
```